app/scripts/controllers/landingPage.js

```diff
@@ -51,10 +52,21 @@ angular.module('openshiftConsole')
 
     AuthService.withUser().then(function() {
       var includeTemplates = !_.get(Constants, 'ENABLE_TECH_PREVIEW_FEATURE.template_service_broker');
-      Catalog.getCatalogItems(includeTemplates).then(function(items) {
+      Catalog.getCatalogItems(includeTemplates).then(_.spread(function(items, errorMessage) {
+        if (errorMessage) {
+          var alert = {
+            name: 'error-loading-catalog-items',
+            data: {
+              type: 'warning',
```

Comment:
Hm, `errorNotification` should default to true.

https://github.com/openshift/origin-web-common/blob/master/src/services/dataService.js#L950